management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResourcesV2.java

```diff
@@ -122,14 +123,15 @@ private String getParallelismName(RepairParallelism parallelism) {
     return parallelism != null ? parallelism.getName() : null;
   }
 
-  private String getRingRangeString(List<RingRange> associatedTokens) {
+  @VisibleForTesting
+  String getRingRangeString(List<RingRange> associatedTokens) {
     if (associatedTokens != null && !associatedTokens.isEmpty()) {
       StringBuilder sb = new StringBuilder();
       for (RingRange ringRange : associatedTokens) {
         sb.append(toRangeString(ringRange)).append(",");
       }
       // remove trailing comma
-      return sb.substring(0, sb.length() - 2);
+      return sb.substring(0, sb.length() - 1);
```

Comment:
Suggestion: for some reason I'm more comfortable with something that looks like this:

```
return associatedTokens.stream().map(i -> toRangeString(i)).collect(Collectors.joining(","));
```

This is stylistic, but I feel like it avoids a two step process and some of the off-by-one issues we've just seen. Up to you.

Reply:
will do